content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -157,6 +157,12 @@ Starting with v1.22 and later, when creating a cluster with kubeadm, if the user
 the `cgroupDriver` field under `KubeletConfiguration`, kubeadm defaults it to `systemd`.
 {{< /note >}}
 
+In Kubernetes {{< skew currentVersion >}}, with the `KubeletCgroupDriverFromCRI`
```

Comment:
Suppose we are in v1.29 now, the above line will be rendered into something like
"In Kubernetes 1.29, with the `KubeletCgroupDriverFromCRI`".
Is this statement still appropriate?
Suppose we are in v1.34 now and the feature gate has been GA'ed and removed,
the above line will be rendered into something like
"In Kubernetes 1.34, with the `KubeletCgroupDriverFromCRI`".
Is this statement still appropriate?

Clearly stating that this feature is in Alpha starting 1.28 is good enough.
Why do we use the `skew` shortcode the make this release number a moving target?

Reply:
I'm indifferent on this, can change it to v1.29. This (`In Kubernetes {{< skew currentVersion >}}`) was something that @sftim suggested in one of his earlier comments so I took it from there.

Reply:
>  Suppose we are in v1.34 now and the feature gate has been GA'ed and removed,

When we change Kubernetes, we should remove the docs. If we don't trust ourselves to do that, I would focus on fixing that lack of trust.

If we could trust the release docs process, I think this change would be fine.

Reply:
As a contributor I'm inclined to agree with @sftim about the docs process here. Especially with a KEP involved, the process should ensure that the documentation is changed when the feature is moved to beta and later GA (or removed before graduating further). WDYT @tengqm?

Reply:
Per https://github.com/kubernetes/website/pull/42160#issuecomment-1665322332, let's follow the path of least risk to the release, at least for now.

Reply:
K, I changed this now to `In Kubernetes v1.28, ...` (not to `Starting with v1.28 ...` as per https://github.com/kubernetes/website/pull/42160#discussion_r1274196798)

PTAL @sftim @tengqm